config/initializers/omniauth.rb

```diff
@@ -29,6 +29,34 @@
   ("#{dfe_sign_in_issuer_uri}:#{dfe_sign_in_issuer_uri.port}" if dfe_sign_in_issuer_uri.present?)
 }
 
+tid_sign_issuer = ENV["TID_SIGN_IN_ISSUER"]
+tid_sign_in_secret = ENV["TID_SIGN_IN_SECRET"]
+tid_sign_in_endpoint = ENV["TID_SIGN_IN_API_ENDPOINT"]
+tid_base_url = ENV["TID_BASE_URL"]
+
+tid_sign_in_endpoint_uri = URI(tid_sign_in_endpoint)
```

Comment:
We should check if `tid_sign_in_endpoint` is present before passing it to `URI` as it cannot take `nil` as an argument - it's the reason why the build is currently failing

```
#44 [web 19/20] RUN DFE_SIGN_IN_API_CLIENT_ID=   DFE_SIGN_IN_API_SECRET=   DFE_SIGN_IN_API_ENDPOINT=   ADMIN_ALLOWED_IPS=   ENVIRONMENT_NAME=   SUPPRESS_DFE_ANALYTICS_INIT=   bundle exec rake assets:precompile
#44 3.738 rake aborted!
#44 3.739 ArgumentError: bad argument (expected URI object or URI string)
#44 3.739 /app/config/initializers/omniauth.rb:37:in `<main>'
```